src/notation/internal/notationactioncontroller.cpp

```diff
@@ -737,6 +737,38 @@ void NotationActionController::padNote(const Pad& pad)
     if (currentNotationElements()->msScore()->inputState().usingNoteEntryMethod(engraving::NoteEntryMethod::RHYTHM)) {
         playSelectedElement();
     }
+
+    std::string spokenNote;
+    switch (pad) {
+    case Pad::NOTE1:
+        spokenNote = "Whole note";
```

Comment:
The names need to be made translatable, for example like this:
```suggestion
        spokenNote = trc("notation", "Whole note");
```

Reply:
@cbjeukendrup Updated.